### PR TITLE
Migrations, Index, and Index Test, no styling

### PR DESCRIPTION
**Pull request by big-sherry**
Migrations, Index, and Index Test, no styling

### CODE REVIEW

**Comment by big-sherry** [comment]
We had already done some work on the show page because we did not want to do a PR while our team was separated. We will be working at doing a PR for each step after this. If this is not sufficient let us know and we can work on creating a new branch and deleting the extra work on this branch before doing a PR.

File: app/javascript/react/components/TrailTile.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const TrailsTile = (props) => {
+    return (
+        <div>
+            <ul>
+                <li>
+                    <Link to={`/trails/${props.id}`}> {props.name} </Link>
+                    {`${props.city} ${props.zip}`}
+                </li>
+            </ul>
```
Comment (dovakeen118) [inline app/javascript/react/components/TrailTile.js]: lots of extra HTML tags here - looks like we only need a `li` - a `ul` would be used for a collection of list items, and the div is not needed as the `li` is the top most element returned

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,25 @@
...
+trail_2 = Trail.create(
+    name: "Long, Long Trail", 
+    city: "West Stix", 
+    zip: "01235", 
+    route_type: "Winding Roads", 
+    length: "2km",
+    elevation: "100m",
+    difficulty: "3",
+    description: "You wil be lost but happy."
+)
```
Comment (dovakeen118) [inline db/seeds.rb:29]: find_or_create_by is great to ensure if you run the seeds file multiple times you won't get duplicate data

File: spec/features/controllers/trails_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+# verb / action 
+# check the response status 
+# check the response content type 
+# check the attributes of the response body (dont forget the use JSON)
+
```
Comment (dovakeen118) [inline spec/features/controllers/trails_controller_spec.rb]: remove code comments!

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+  root 'static_pages#index'
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get '/trails', to: "static_pages#index"
+  get '/trails/:id', to: "static_pages#index"
+  
+  namespace :api do
+    namespace :v1 do
+      resources :trails, only: [:index, :show]
```
Comment (dovakeen118) [inline config/routes.rb]: we should remove the references to `show` here for now (lines 7 & 11)

File: app/javascript/react/components/App.js [hunk 5]
```diff
@@ -1,7 +1,21 @@
...
-  return (<h1>Make It So React</h1>)
+
+const App = (props) => {
+  return (
+    <div>
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/" component={TrailsIndexContainer} />
+          <Route exact path="/trails" component={TrailsIndexContainer} />
+          <Route exact path="/trails/:trailId" component={TrailShowContainer} />
```
Comment (dovakeen118) [inline app/javascript/react/components/App.js]: let's remove references of show for now!

File: app/controllers/api/v1/trails_controller.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+    def index
+        trails = Trail.all
+        render json: trails
+    end
+
+    def show
+        trail = Trail.find(params[:id])
+        render json: trail
+    end
```
Comment (dovakeen118) [inline app/controllers/api/v1/trails_controller.rb]: let's remove for now!

**Review by dovakeen118 — COMMENTED** [review]
looking good! some comments to clean up parts of the code, and in making other changes we should remove other references to `show` for now so that the PR is in scope :)

**Comment by big-sherry** [comment]
I believe we have cleaned up the code and removed all references to show.

**Pull request merged**